internal/driver/config/opl_config_namespace_watcher.go

```diff
@@ -33,7 +36,14 @@ type (
 	}
 )
 
-var _ namespace.Manager = (*oplConfigWatcher)(nil)
+var (
+	_        namespace.Manager = (*oplConfigWatcher)(nil)
+	cache, _                   = ristretto.NewCache(&ristretto.Config{
+		MaxCost:     20_000_000, // 20 MB max size, each item ca. 10 KB => max 2000 items
+		NumCounters: 20_000,     // max 2000 items => 20000 counters
+		BufferItems: 64,
+	})
```

Comment:
Ristretto config is a science for itself. I recommend this:

```suggestion
	cache, _                   = ristretto.NewCache(&ristretto.Config{
		NumCounters:        maxItems * 10, // Counters should be 10x max cost (see ristretto docs).
		MaxCost:            maxItems,
		BufferItems:        64,
		Metrics:            true, // consider false
		IgnoreInternalCost: true,
		Cost: func(value interface{}) int64 {
			return 1
		},
 	})
``` 

And then set `maxItems` to `20MB / 10kb = 2048`. Although I'd argue that up to 500MB is fine for this cache.

Reply:
Is this better, though? OPL definitions are not all the same size, yet their individual cost is easy to calculate.

LMK.

Reply:
ok, we can also fine-tune this by checking how this is doing in our tracing